bin/sqlline.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+"""sqlline.py"""
```

Comment:
This was to quieten  PEP8 from complaining about "Missing docstring in public module". But, I'll remove it since uniform license header is probably more important.